Baikal/Controllers/clw_scene_controller.h

```diff
@@ -95,9 +93,13 @@ namespace Baikal
         // If scene attributes changed
         void UpdateSceneAttributes(Scene1 const& scene, Collector& tex_collector, ClwScene& out) const override;
 
+    private:
         // Update intersection API
         void UpdateIntersector(Scene1 const& scene, ClwScene& out) const;
         void UpdateIntersectorTransforms(Scene1 const& scene, ClwScene& out) const;
+        // Clear intersector and load meshes into it.
+        void ReloadIntersector(Scene1 const& scene, ClwScene& inout) const;
```

Comment:
What is the reason to move it to private from protected?

Reply:
Because of better encapsulation. And we don't use protection-features in here